src/plugins/discover/public/application/components/discover_grid/discover_grid_flyout.tsx

```diff
@@ -150,17 +152,20 @@ export function DiscoverGridFlyout({
               </EuiFlexItem>
             )}
             {activePage !== -1 && (
-              <EuiFlexItem grow={false}>
-                <EuiPagination
-                  aria-label={i18n.translate('discover.grid.flyout.documentNavigation', {
-                    defaultMessage: 'Document navigation',
-                  })}
-                  pageCount={pageCount}
-                  activePage={activePage}
-                  onPageClick={setPage}
-                  compressed
-                />
-              </EuiFlexItem>
+              <EuiHideFor sizes={['xs', 's', 'm']}>
```

Comment:
So we would lock out with smaller - medium screen. So for those who can't who can't just make the browser window bigger, before they would need to buy a new machine (I'm sure they would :),  what do you think just hiding `euiPagination__compressedText` for `xs`, `s`, `m` sizes, so it wouldn't display the `of` ?

Reply:
I'm good with that. It does look odd and it ought to ultimately be addressed in EUI, in my opinion. I'll adjust this PR to do as you suggest.